lib/fluent/plugin/out_vmware_loginsight.rb

```diff
@@ -22,13 +22,13 @@ class ConnectionFailure < StandardError; end
       Fluent::Plugin.register_output('vmware_loginsight', self)
 
       ### Connection Params ###
-      config_param :scheme, :string, :default => 'http'
+      config_param :scheme, :string, :default => 'https'
```

Comment:
Though we should use `https`,  changing default would affect some users. We haven't published release notes, perhaps we can do that, or we can delay this change for some other day.